src/libraries/System.Private.CoreLib/src/System/StringComparer.cs

```diff
@@ -318,7 +318,7 @@ public override int Compare(string? x, string? y)
 
             if (_ignoreCase)
             {
-                return string.Compare(x, y, StringComparison.OrdinalIgnoreCase);
+                return System.Globalization.Ordinal.CompareStringIgnoreCase(ref x.GetRawStringData(), x.Length, ref y.GetRawStringData(), y.Length);
```

Comment:
Is `System.Globalization.` needed? I see `using System.Globalization;` in the header. Or it clashes with something?

Reply:
There's an Ordinal property on this type, so just writing "Ordinal" conflicts.  I could have just done "Globalization.Ordinal", but for whatever reason I've never liked leaving off part but not all of a namespace.